web-ng/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "lib": [
+			"es2018",
+			"dom"
```

Comment:
Looks like this is using spaces instead of tabs. Is there a linter that would detect or fix this? Or are .json not linted?

Reply:
No, I think you meant this is using tabs instead of spaces. I'll change it to spaces. The tslint.json generated by gts excluded all the .json files.

Reply:
Sorry, that's what I meant. Thanks for fixing it!